.github/workflows/maven-cd.yml

```
@@ -3,6 +3,12 @@
 name: maven-cd
 on:
   workflow_call:
+    inputs:
+      java-major-version:
```

major isn't important here imo, when people talk about a java version they 99% of the time say Java 17, not 17.1.22, e.g. see in pipeline library docs, it says version not major version: https://github.com/jenkins-infra/pipeline-library#optional-arguments

```suggestion
      java-version:
```